pkg/network/render.go

```diff
@@ -62,6 +62,13 @@ func Render(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.BootstrapResult
 	}
 	objs = append(objs, o...)
 
+	if conf.Migration != nil {
+		o, progressing, err = renderCRDForMigration(conf, manifestDir)
```

Comment:
could you add a comment explaining why we need to render the new CRDs during migration?

Reply:
fixed

Reply:
thanks! the comment looks good.
Can you move the comment to the first commit and extract egress firewall migration to a separate PR? I think it would be good to get some more eyes on that.

Reply:
Done.